### PR TITLE
Update dart_vlc.podspec to account for new runner path

### LINKED ISSUE #231: error: install_name_tool: can't open file
Out of nowhere I'm unable to build for macOS any more. I keep getting the following error no matter what I try:
```
error: install_name_tool: can't open file: /Users/groovinchip/dev/flutter_projects/packages_and_plugins/stream-chat-flutter/packages/stream_chat_flutter/example/macos/Pods/VLCKit/VLCKit.framework/VLCKit (No such file or directory)
```
I've done the following to no avail:
* `flutter clean`
* `pod update`
* `pod repo update`
* Completely regenerating the `macos` runner
* Removing the `dart_vlc` dependency from `pubspec.yaml`, running `flutter pub get` to remove it completely, and then re-adding `dart_vlc`

I'm out of ideas and need some help.

It's worth noting that my Pods directory looks like this:
Screen Shot 2022-03-21 at 12 21 05 PM
It seems like the path that the VLC macOS podspec expects is wrong?

### PR DESCRIPTION
**Pull request by GroovinChip**
The Flutter macOS runner has been updated with a new path structure in /macos/Pods/, so I have updated `dart_vlc.podspec` to account for this. I have tested this with the example app and it works.

Closes #231

### CODE REVIEW

**Comment by alexmercerind** [comment]
Thanks! I've sent the repository collaborator invitation to you, feel free to merge or commit changes.
I think it will be helpful to you / your organization in make changes according to your requirement & in-turn it will be helpful for plugin's development as-well.
I will make some improve `dart_vlc` soon & want to rewrite things ~~a bit~~ here & there.

**Comment by GroovinChip** [comment]
I saw the invite, thank you very much! This will indeed be quite helpful for my organization. I'm gonna see what I can do about why release builds for macos are failing before I merge. 

Once I do merge, though, are you ok with me publishing the changes to pub.dev? Just want to make sure I don't overstep :)

**Comment by alexmercerind** [comment]
> I saw the invite, thank you very much! This will indeed be quite helpful for my organization. I'm gonna see what I can do about why release builds for macos are failing before I merge.
> 
> Once I do merge, though, are you ok with me publishing the changes to pub.dev? Just want to make sure I don't overstep :)

Thanks! Yeah, I'm perfectly fine with you making new releases. Just make sure, that you fetch the sub-modules present in the repository [here](https://github.com/alexmercerind/dart_vlc/tree/master/dartvlc/external). These are required on Windows & Linux as of now.

If you have already cloned the repository, I think following command will work (I'm not exactly `git` geek & expect no-one to be, so sharing in advance):
```
git submodules update --recursive --remote
````
or if you freshly clone, adding --recursive flag will ensure that all the submodules are correctly fetched.
```
git clone dart_vlc_repository --recursive
```

TL;DR

Just ensure that [this](https://github.com/alexmercerind/dart_vlc/tree/master/dartvlc/external/dart_api) & [this](https://github.com/alexmercerind/dart_vlc/tree/master/dartvlc/external/libvlcpp) folder isn't empty before publishing to pub.dev.

**Comment by GroovinChip** [comment]
Sounds good!

I think the problem with the CI is related to using Flutter's `master` channel, because everything works fine for me on stable. I'm gonna call this good to merge.

**Comment by alexmercerind** [comment]
> Sounds good!
> 
> I think the problem with the CI is related to using Flutter's `master` channel, because everything works fine for me on stable. I'm gonna call this good to merge.

When I was initially working on the plugin, nearly nothing was available on `stable` channel for the desktop, so having `master` channel for texture support etc. I think that's not a necessity anymore & switching CI to `stable` will be good.

**Comment by alexmercerind** [comment]
I'd appreciate mentioning two other people in the changelog :smile::
48a14408c5474dd94f86bb1ec2f2c16dc1b3f18a
48a14408c5474dd94f86bb1ec2f2c16dc1b3f18a

Thanks!

**Comment by GroovinChip** [comment]
Done!
